cmd/kubeadm/app/phases/upgrade/staticpods.go

```diff
@@ -227,7 +232,7 @@ func upgradeComponent(component string, waiter apiclient.Waiter, pathMgr StaticP
 	fmt.Printf("[upgrade/staticpods] Moved new manifest to %q and backed up old manifest to %q\n", currentManifestPath, backupManifestPath)
 
 	if waitForComponentRestart {
-		fmt.Println("[upgrade/staticpods] Waiting for the kubelet to restart the component")
+		fmt.Printf("[upgrade/staticpods] Waiting for the kubelet to restart the component. This operation will timeout in %v\n", UpgradeManifestTimeout)
 
```

Comment:
@xlgao-zju sorry I forgot to clck add comment 😶
Please make this more consistent with other kubeadm Wait messages e.g.
```
fmt.Printf("[upgrade/staticpods] Waiting for the kubelet to restart the component")
fmt.Printf("[upgrade/staticpods] This might take a minute or longer depending on the component/version gap (timeout %v\n", UpgradeManifestTimeout)
```
This includes a short reassuring description about what Kubeadm is doing and why the user should wait (in this case I suggest a generic note)